Package.swift

```diff
@@ -18,7 +18,7 @@ let package = Package(
         .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0-beta.2"),
         .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0-beta.2"),
         .package(url: "https://github.com/vapor/fluent-postgres-driver.git", from: "2.0.0-beta.2"),
-        .package(url: "https://github.com/vapor/fluent-mysql-driver.git", from: "4.0.0-beta.2"),
```

Comment:
It should not affect tests. it'd be better to uncomment it and change back swift-tools-version to 5.1

Reply:
Tried, just for test